orchagent/bulker.h

```diff
@@ -372,31 +355,22 @@ class EntityBulker
                         ts.push_back(attr);
                         status_vector.push_back(object_status);
                     }
+
+                    if (rs.size() >= MAX_BULK_SIZE)
+                    {
+                        flush_setting_entries(rs, ts, status_vector);
+                        rs.clear();
+                        ts.clear();
+                        status_vector.clear();
```

Comment:
If you use reference in above function, you can move the 3 clear() into the above function.

Reply:
Moved the clear() into the function.